<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="urdf1">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-6.0729E-05 9.277E-05 0.044384"
        rpy="0 0 0" />
      <mass
        value="5.0287" />
      <inertia
        ixx="0.01679"
        ixy="-1.5103E-05"
        ixz="6.7539E-06"
        iyy="0.016804"
        iyz="-1.0317E-05"
        izz="0.028424" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="Link1">
    <inertial>
      <origin
        xyz="-2.04097449292562E-05 0.0113015511019697 0.0655223391033708"
        rpy="0 0 0" />
      <mass
        value="55.7444455513424" />
      <inertia
        ixx="0.727810823126981"
        ixy="-6.08303988595912E-06"
        ixz="-2.12903000814142E-07"
        iyy="0.927554282928859"
        iyz="0.0220552256243757"
        izz="1.04521065780087" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint1"
    type="continuous">
    <origin
      xyz="0 0 0.3645"
      rpy="3.1416 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link1" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="Link2">
    <inertial>
      <origin
        xyz="-1.2631E-05 0.009987 0.099695"
        rpy="0 0 0" />
      <mass
        value="63.019" />
      <inertia
        ixx="0.76516"
        ixy="4.9711E-06"
        ixz="-5.2419E-06"
        iyy="0.96484"
        iyz="0.022055"
        izz="1.1145" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint2"
    type="continuous">
    <origin
      xyz="0 0.6115 0"
      rpy="1.5708 1.5708 0" />
    <parent
      link="Link1" />
    <child
      link="Link2" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="Link3">
    <inertial>
      <origin
        xyz="2.22 -1.9964E-13 -0.55613"
        rpy="0 0 0" />
      <mass
        value="226.93" />
      <inertia
        ixx="2.9105"
        ixy="-9.9422E-06"
        ixz="1.3858E-14"
        iyy="27.692"
        iyz="7.0728E-15"
        izz="28.391" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint3"
    type="continuous">
    <origin
      xyz="0 0 0"
      rpy="1.5708 0 0" />
    <parent
      link="Link2" />
    <child
      link="Link3" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="Link4">
    <inertial>
      <origin
        xyz="7.9595E-05 2.4959 -0.0035265"
        rpy="0 0 0" />
      <mass
        value="212.87" />
      <inertia
        ixx="27.578"
        ixy="-0.00024036"
        ixz="0.0011073"
        iyy="2.9769"
        iyz="-9.7846E-05"
        izz="28.074" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint4"
    type="continuous">
    <origin
      xyz="4.44 0 -0.035"
      rpy="-3.1416 0 1.5708" />
    <parent
      link="Link3" />
    <child
      link="Link4" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="Link5">
    <inertial>
      <origin
        xyz="1.2631E-05 -0.099695 -0.009987"
        rpy="0 0 0" />
      <mass
        value="63.019" />
      <inertia
        ixx="0.76516"
        ixy="-5.2419E-06"
        ixz="4.9711E-06"
        iyy="1.1145"
        iyz="0.022055"
        izz="0.96484" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link5.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link5.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint5"
    type="continuous">
    <origin
      xyz="0 5.0815 0"
      rpy="-1.5708 0 0" />
    <parent
      link="Link4" />
    <child
      link="Link5" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="Link6">
    <inertial>
      <origin
        xyz="-2.041E-05 0.065522 -0.011302"
        rpy="0 0 0" />
      <mass
        value="55.744" />
      <inertia
        ixx="0.72781"
        ixy="-2.129E-07"
        ixz="6.083E-06"
        iyy="1.0452"
        iyz="-0.022055"
        izz="0.92755" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link6.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link6.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint6"
    type="continuous">
    <origin
      xyz="0 -0.6115 0"
      rpy="1.5708 0 0" />
    <parent
      link="Link5" />
    <child
      link="Link6" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="Link7">
    <inertial>
      <origin
        xyz="8.7299E-06 -0.00011054 -0.069201"
        rpy="0 0 0" />
      <mass
        value="5.0287" />
      <inertia
        ixx="0.016781"
        ixy="-2.5869E-06"
        ixz="9.7088E-07"
        iyy="0.016813"
        iyz="-1.2293E-05"
        izz="0.028424" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link7.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf1/meshes/Link7.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint7"
    type="continuous">
    <origin
      xyz="0 0.38932 0"
      rpy="-1.5708 0 0" />
    <parent
      link="Link6" />
    <child
      link="Link7" />
    <axis
      xyz="0 0 1" />
  </joint>
</robot>